src/assets/database.sql

```diff
@@ -91,6 +92,16 @@ INSERT INTO `commands` (`id`, `name`, `description`, `examples`, `tool`, `type`,
 (96, 'Source addresses (list separated by +)', NULL, NULL, 16, 'input', '-t source', 'Example: 203.0.113.0/24+198.51.100.0/24', NULL, NULL),
 (97, 'Destination addresses (list separated by +)', NULL, NULL, 16, 'input', '-t destination', 'Example: 192.0.2.0/24+198.18.0.0/15', NULL, NULL);
 
+--
+-- Populate data for `commands`: hping3
+--
+
+INSERT INTO `commands` (`id`, `name`, `description`, `examples`, `tool`, `type`, `command`, `value`, `example`, `sudo`, `category`) VALUES
+(98, 'Packet count', 'Number of packages to send', 'hping3 --count 10', 18, 'input', '--count', '10', '10', NULL, 'COUNTER'),
+(99, 'Interval', 'Wait X microseconds', 'hping3 --interval u1000', 18, 'input', '--interval', NULL, 'u10000', NULL, 'TIMER'),
+(100, 'Debug', 'Debbuging info', 'hping3 --debug', 18, 'checkbox', '--debug', NULL, NULL, NULL, 'DEBUG'),
+(101, 'Data size', NULL, 'hping3 --data 100', 18, 'input', '--data', NULL, '0', NULL, 'SIZE');
```

Comment:
same here, can we change the style from having the value in a different field to have they all together?

Reply:
Same as before... we want to set it by default but leave user capable of changing it.